### ISSUE TITLE
I cant see an option to customize

### ISSUE THREAD

**Issue by owlreddy** [pop-os/cosmic-dock#26]
The Dash to Dock settings by clicking on the 9 dot's icon is not available. I just updated to PopOS 21.04 & in my Dock the 9 dot's icon is absent and I cant find a Dash to Dock settings option anywhere( which I can see in the Pic in Readme)

**Comment by rubenjr0** [comment]
Same here, I can't find a way to customize it aside from the options provided in the desktop menu

**Comment by NathanaelA** [comment]
Yeah, it appears they were trying to make it "simpler", but this creates the inability to make certain changes.    You can use `dconf-editor` and then navigate to 'org.gnome.shell.extensions.dash-to-dock' and then you can see all the settings.

**Comment by bertjeduyck** [comment]
same here, no settings and no icons either unless these icons are correct.

![image](https://user-images.githubusercontent.com/60874591/126139253-01e0f011-581a-48bb-8ff9-5c190e48d00d.png)
